core/src/main/java/org/nervos/ckb/methods/type/Header.java

```diff
@@ -26,17 +25,15 @@
      * "version": 0
      */
 
-    @JsonProperty("cellbase_id")
-    public String cellbaseId;
 
     public String difficulty;
     public String hash;
-    public int number;
+    public String number;
```

Comment:
Please update the comment above as `number` is not a string.

Reply:
OK